src/views/List.jsx

```diff
@@ -24,36 +26,57 @@ export function List({ data, listToken }) {
 
 	return (
 		<>
-			<p>
-				Hello from the <code>/list</code> page!
-			</p>
+			{/* The height of the green section of the navigation tab is: h-20 (5rem);
+		The button is sticking out by -mt-6 (1.5rem);
+		You'll need 6.5rem of empty space at the bottom of your page so that the user can see their entire list.  */}
 			{!data.length ? (
-				<div>
-					<p>Your shopping list is currently empty.</p>
-					<button onClick={routeToAddItem}>Add your first item</button>
+				<div className="flex flex-col items-center gap-2">
```

Comment:
@jennaly adding padding to the top so that it's seperated from the header.

```suggestion
				<div className="flex flex-col items-center gap-2 mt-32">
```